### DISCUSSION TITLE
How to make a standalone input?

### DISCUSSION THREAD

**Discussion by NicolasSacC** [filamentphp/filament#4340] [Help] [answered]
Sometimes, I have some inputs that aren't really in a form. They are maybe in the middle of another component or something like that. I want to keep a consistent design on my website, so I would like to create some standalone inputs that would sync to the Livewire component the same way a form would do without having to generate a form.

Is this doable? What else would be the best practice in this case?

For examples, I am building a "Cart" for my web app. The users must be able to change the number of products easily using an input type "number". How would this be done using Filament inputs? I could make a Livewire component for each element, but this seems a bit overkill since it would just pass the data to the parent component...

**Comment by zepfietje** [comment, marked as answer]
Hey @NicolasSacC!

Extracting Blade components (like the text input) in order to use them standalone is on our radar: https://github.com/filamentphp/filament/issues/4164. 🙂
